src/components/dropdown/menu.tsx

```diff
@@ -127,12 +127,17 @@ class Menu<T> extends Component<Props<T>> {
             padding
           )
 
+          const key =
+            item.key || typeof item.value === "object"
+              ? `item-${index}`
+              : item.value?.toString()
```

Comment:
The ternary isn't equivalent to what we did before. Consider the case of a `null` value. Since the type of null is an object it would result of `undefined` being returned as key which would result in another error.
Wouldn't this do the trick:
```suggestion
          const key = item.key || (item.value !== "object" && item.value?.toString()) || `item-${index}`
```

Reply:
You are right, sorry. Close, you missed the `typeof`

item.key || (typeof item.value !== "object" && item.value.toString()) || `item-${index}`

Thanks for the help!